scripts/cem-plugin-react-wrapper.js

```diff
@@ -175,8 +175,10 @@ export default function genReactWrapper({
             const fileImports = modules
                 .filter(
                     (m) =>
-                        m.declarations.length === 0 &&
-                        m.path.indexOf('sync/') === -1
+                        m.exports.length === 1 &&
```

Comment:
If we used [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) would we be able to simplify to a single condition?